src/character.h

```diff
@@ -67,6 +65,7 @@ struct bionic;
 struct construction;
 struct dealt_projectile_attack;
 struct display_proficiency;
+class known_magic;
```

Comment:
would you mind putting this within the list of other classes? you also appear to be missing an `enchantment` forward declaration.

Reply:
enchantment is forward declared in type_id.h